react/typings/vtex.styleguide.d.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+declare module 'vtex.styleguide' {
+  export const ToastContext: React.Context<{
+    showToast: (message: string) => void
+    toastState: { isToastVisible: boolean }
+  }>
+}
```

Comment:
question (non-blocking): just out of curiosity, how hard is to implement this toast typing on styleguide?

Reply:
I'd need to rewrite the component in typescript